addon/templates/frost-list-core.hbs

```diff
@@ -22,16 +22,12 @@
         lastReached=loadNext
         as |record index|
       }}
-        {{#if hasBlock}}
-          {{yield record index}}
-        {{else}}
-          {{component item
-            model=record
-            hook=(concat hook '-item-' index)
-            hookQualifiers=(hash index=index)
-            onSelect=(action 'selectItem')
-          }}
-        {{/if}}
+        {{component item
+          model=record
+          hook=(concat hook '-item-' index)
```

Comment:
If you have a hookQualifier below with the index, why concat the index on the hook?